submissions/aymkin/OOP-frogger/app.js

```diff
@@ -0,0 +1,121 @@
+let scoreCount = 0;
+
+const START = {
+	x: 202,
+	y: 405,
+};
+
+const FIELD = {
+	column: 101,
+	row: 83,
+	leftBorder: 101,
+	rightBorder: 303,
+	topBorder: 0,
+	bottomBorder: 322,
+};
+
+const SPEED = (min = 75, max = 200) =>
+	Math.floor(Math.random() * (max - min + 1)) + min;
+
+const Creature = function (x, y) {
+	this.x = x;
+	this.y = y;
+};
+
+
+	const Enemy = function(x, y, speed, gamer) {
+	Creature.call(this, x, y);
+	this.speed = speed;
+	this.gamer = gamer;
+	this.sprite = 'images/enemy-bug.png';
+};
```

Comment:
The variable `sprite` is the same for classes `Enemy` and `Player`, it can be declared in a general class `Creature`, and the value should be added to a constant.

Reply:
@madmaxWMFU  
I believe the sprites are different, aren't they?

![enemy](http://i.imgur.com/2WRLBZL.png)

![enemy](http://i.imgur.com/jtQ5JLC.png)

Reply:
```javascript
const playerStartValues = {
	x: 202,
	y: 405,
        img: 'images/char-boy.png'
};

const enemyStartValues = {
	x: 0,
	yFirsrtEnemy: ,
        ySecondEnemy: ,
        yThirdEnemy: ,
        img: 'images/enemy-bug.png',
};
const Creature = function (x, y, sprite) {
	this.x = x;
	this.y = y;
        this.sprite = sprite;
};

const Enemy = function(x, y, sprite, gamer) {
	Creature.call(this, x, y, sprite);
	this.speed = this.getSpeed();
	this.gamer = gamer;
};

const Player = function(x, y, sprite) {
	Creature.call(this, x, y, sprite);
};

const player = new Player(playerStartValues.x, playerStartValues.y, playerStartValues.img);
const allEnemies = [enemyStartValues.yFirsrtEnemy, enemyStartValues.ySecondEnemy, enemyStartValues.yThirdEnemy].map(y =>  new Enemy(enemyStartValues.x, y, enemyStartValues.img, player));

```

Reply:
the property is the same and the value is different

Reply:
all done, now looks much better
Thank you a lot